[split cat fx passes] Normalize squeeze

devashishshankar opened the pull request:
Summary: Sometimes, squeeze can be a "call_method" instead of a "call_function". Normalizing it will make it amenable to pattern matching by passes like "split->squeeze"

Test Plan: * CI tests

Differential Revision: D46031846



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10

devashishshankar commented:
@pytorchbot label "topic: not user facing"

jansel reviewed (changes requested):
Test failures seem real

devashishshankar commented:
> Test failures seem real

@jansel fixed

devashishshankar committed:
normalize squeeze (#102294)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/102294

Sometimes, squeeze can be a "call_method" instead of a "call_function". Normalizing it will make it amenable to pattern matching by passes like "split->squeeze"

Test Plan: * CI tests

Reviewed By: jansel

Differential Revision: D46031846

fbshipit-source-id: 7ef0d4f65f4edcc9e7269fbb9ac204fcba6a1eee
